### ISSUE TITLE
[a11y] md-tabs has twice as many tabs as it should

### ISSUE THREAD

**Issue by jonathantyates** [angular/material#9450]
**Actual Behavior**:
 -  `What is the issue? *`  There are twice as many role="tab" inside md-tabs as there should be.  This causes ChromeVox to read "Tab 2 of 4" with only 2 tabs.
 -  `What is the expected behavior?`  It should read tab 2 of 2

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:` http://codepen.io/anon/pen/zBgVmJ
 -  `Details:` Turn on ChromeVox and focus a tab.

**Angular Versions**: *
 -  `Angular Version:` 1.5
 -  `Angular Material Version:` 1.1.0

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` 52
 -  `OS: *` Ubuntu
 -  `Stack Traces:`

**Comment by clshortfuse** [comment]
`md-tabs-dummy-wrapper` might need a `[hidden]` attribute so ChromeVox will properly ignore it.

**Comment by crisbeto** [comment]
Either that, or removing the `role="tab"` altogether.

**Comment by marcysutton** [comment]
That solution was in there for touch devices. Make sure you test it there too (and keep in mind ChromeVox is not a popular screen reader for people with disabilities).
